spec/functional/command/repo/push_spec.rb

```diff
@@ -109,7 +109,7 @@ module Pod
     end
 
     it 'refuses to push if --no-overwrite is passed and the spec exists' do
-      cmd = command('repo', 'push', 'master', 'JSONKit.podspec', '--no-overwrite')
+      cmd = command('repo', 'push', 'master', 'JSONKit.podspec', '--commit-message="foo"', '--no-overwrite')
```

Comment:
This should probably be 2 tests (one with `--commit-message` and one without), since if it was done like that originally the bug would have been caught

Reply:
yeap agreed.

Reply:
@amorde done